### DISCUSSION TITLE
Is CefSharp affected by the security vulnerabilites CVE-2023-4863, CVE-2023-5217

### DISCUSSION THREAD

**Discussion by daveMueller** [cefsharp/CefSharp#4601] [Q&A] [answered]
Hi guys,

as we use CefSharp in one of our products it would be great if somebody could tell me if the library is also affected by the security vulnerabilities CVE-2023-4863, CVE-2023-5217.

https://nvd.nist.gov/vuln/detail/CVE-2023-4863
https://nvd.nist.gov/vuln/detail/CVE-2023-5217

Thanks a lot for any info ☺️.

**Comment by campersau** [comment, marked as answer]
Looks like it is, see "Security" Tab:
https://github.com/cefsharp/CefSharp/security/advisories/GHSA-j646-gj5p-p45g
https://github.com/cefsharp/CefSharp/security/advisories/GHSA-4c29-gfrp-g6x9

**Comment by daveMueller** [comment]
Thanks a lot for the quick feedback 🙏 . As far as I understand the issue seems to fixed in the newest versions of `libvpx ` and `libwebp`. Does anybody know if a fix for CEF and CefSharp is already planned?

**Reply by campersau** [reply]
You can see that when you follow the links:

https://github.com/cefsharp/CefSharp/security/advisories/GHSA-j646-gj5p-p45g
Patched versions: 116.0.230

https://github.com/cefsharp/CefSharp/security/advisories/GHSA-4c29-gfrp-g6x9
Patched versions: 117.2.20

**Reply by daveMueller** [reply]
Thanks a lot @campersau 🙏
